tests/integration/docker/volume.bats

```diff
@@ -48,6 +48,31 @@ setup() {
 	[ $RC -eq 0 ]
 }
 
+@test "Volume - bind-mount a directory" {
+	dir_path="$(pwd)/sharedSpace"
+	test_file="foo"
+	containerPath="/root/sharedSpace"
+	mkdir "$dir_path"
+	touch "$dir_path/$test_file"
+	$DOCKER_EXE run -i -v "$dir_path":"$containerPath" busybox ls "$containerPath" | grep "$test_file"
+	RC=$?
+	rm -r "$dir_path"
+	$DOCKER_EXE rm $($DOCKER_EXE ps -qa)
+	[ $RC -eq 0 ]
+}
+
+@test "Volume - bind-mount a single file" {
+	test_file="$(pwd)/foo"
+	file_content="bar"
+	echo "$file_content" >  "$test_file"
+	containerPath="/root/foo"
+	$DOCKER_EXE run -i -v "$test_file":"$containerPath" busybox cat "$containerPath" | grep "$file_content"
+	RC=$?
```

Comment:
I don't see how this can work as if the `docker run` fails, the test will immediately abort. I think you need to do something like the following to allow `docker run` to potentially fail but capture the return code and continue:

```
{ $DOCKER_EXEC run ... | grep ...; RC=$?; } || true
rm ...
$DOCKER_EXE rm ...
[ $RC -eq 0 ]
```